web/src/QuestionActions.jsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) [2022] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import Popup from "./Popup";
+import { partition } from "./utils";
+
+/**
+ * Returns given text capitalized
+ *
+ * TODO: make it work with i18n
+ *
+ * @param {String} text - string to be capitalized
+ * @return {String} capitalized text
+ */
+const label = text => `${text[0].toUpperCase()}${text.slice(1)}`;
+
+/**
+ * A component for building a Question actions, using the defaultAction
+ * as the Popup.PrimaryAction
+ *
+ * NOTE: We use the Popup component for displaying a question, wrapping its actions inside a
+ * Popup.Actions component, which must be an immediate child of Popup. That's why we use
+ * React.Fragment (aka <>) here for wrapping the actions instead of directly using the Popup.Actions.
+ *
+ * @param {object} props - component props
+ * @param {Array.<String>} props.actions - the actions to be shown
+ * @param {String} [props.defaultAction] - the action to be shown as primary
+ * @param {function} props.actionCallback - the function to be called when user clicks on action
+ * @param {Object} [props.conditions={}] - an object holding conditions, like when an action is disabled
+ */
```

Comment:
Good docs, but it is not rendered (I'm using `npx jsdoc -r src`) and I have no idea how to fix that

Reply:
I'll try to explain it better later (or maybe to make a PR _fixing_ it), but as far as I can see it is not part of the rendered output because it is using `export default function Component() {}`. The Popup component, for example, is part of the output but it is using a different approach for _making it public_, 

```js
const Popup = () => {}
...
export default Popup
```

A quick test reveled that using the same technique for the QuestionActions includes it in the output. So, I suspect that it is something that can be changed using jsdoc options (I have to read about it).